src/__main__.py

```diff
@@ -49,10 +49,14 @@ async def get_crime(request):
     if crime is None:
         raise Exception("Crime not found")
 
-    return crime
+    return crime.to_dict()
 
+# thread '<unnamed>' panicked at src/server.rs:329:14:
+# called `Result::unwrap()` on an `Err` value: insertion failed due to conflict with previously registered route: /crimes/:crime_id
+# note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
 
-@app.put("/crimes/:crime_id")
+# @app.put("/crimes/:crime_id", auth_required=True) #app will not start
+@app.put("/crimes/:crime_id")   #app will start but requests are rejected as unauthorized
```

Comment:
@rbarbalat , the error occurs as there is already an auth_requried route at line 43

Reply:
@sansyrox, why is the auth_required setting for GET /crimes/:crime_id connected to auth_required setting for PUT /crimes/:crime_id?  Shouldn't you be able to set that setting independently?  I expect that it would often be the case that the PUT route should require authorization but the GET route should not.